src/SFA.DAS.ApprenticeCommitments.Web/Services/OuterApi/IOuterApiClient.cs

```diff
@@ -1,18 +1,21 @@
-﻿using RestEase;
-using System;
-using System.Threading.Tasks;
-
-namespace SFA.DAS.ApprenticeCommitments.Web.Services.OuterApi
-{
-    public interface IOuterApiClient
-    {
-        [Get("/registrations/{id}")]
-        Task<VerifyRegistrationResponse> GetRegistration([Path] Guid id);
-
-        [Post("/registrations")]
-        Task VerifyRegistration([Body] VerifyRegistrationRequest verification);
-
-        [Get("/apprentices/{id}/apprenticeships")]
-        Task<Apprenticeship[]> GetApprenticeships([Path] Guid id);
-    }
+﻿using RestEase;

```

Comment:
Again, the whole file has changed.  Looking at it via CLI (`git show 23a2c81b --ws-error-highlight=all`) I can see that the Line Feed character [has been added](https://stackoverflow.com/a/53359792):
![image](https://user-images.githubusercontent.com/201727/110881077-9e478980-82d7-11eb-887e-fb92041e9e7c.png)

This may be caused by an editor asking "Do you want to fix line endings?", possibly.